bindings_wasm/src/opfs.rs

```diff
@@ -1,13 +1,17 @@
 use futures::FutureExt;
 use std::future::Future;
 use wasm_bindgen::prelude::*;
-use xmtp_db::{OpfsSAHError, OpfsSAHPoolUtil};
+use xmtp_db::{init_sqlite, OpfsSAHError, OpfsSAHPoolUtil};
 
 #[wasm_bindgen]
 pub struct Opfs;
 
 #[wasm_bindgen]
 impl Opfs {
+  pub async fn init_sqlite_opfs() {
```

Comment:
will it ever be necessary to call this function?

Reply:
only if you want to use the opfs utility functions _before_ the client is initialized. I figured there is a possibility the xmtp-js sdk might want to check the capacity/wipe files before creating a client, in which case this function would need to be called first.